app/stac_api/models.py

```diff
@@ -146,9 +156,10 @@ def clean(self):
 
 
 class Collection(models.Model):
-    created = models.DateTimeField(auto_now_add=True)  # datetime
-    updated = models.DateTimeField(auto_now=True)  # datetime
-    description = models.TextField()  # string  / intentionally TextField and
+    collection_name = models.CharField(unique=True, max_length=255, validators=[validate_name])
```

Comment:
maybe keep the comment about the different naming in spec ('id') and model ('name') (bc. 'id' is used )

Reply:
Done